x-pack/plugins/index_management/public/application/components/component_templates/component_template_wizard/component_template_edit/component_template_edit.tsx

```diff
@@ -85,6 +90,38 @@ export const ComponentTemplateEdit: React.FunctionComponent<RouteComponentProps<
       return;
     }
 
+    if (updatedComponentTemplate._meta?.managed_by === 'fleet' && dataStreams.length) {
```

Comment:
nit: would it make sense to extract the string into a const?

Reply:
Yes it will make sense